plugins/performance-lab/includes/site-health/cache-control-headers/helper.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * Helper functions used for Cache-Control headers site health check.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Tests the Cache-Control headers.
+ *
+ * @since n.e.x.t
+ * @access private
+ *
+ * @return array{label: string, status: string, badge: array{label: string, color: string}, description: string, actions: string, test: string} Result.
+ */
+function perflab_cch_check_cache_control_test(): array {
+	$result = array(
+		'label'       => __( 'Cache settings are optimal for site performance.', 'performance-lab' ),
+		'status'      => 'good',
+		'badge'       => array(
+			'label' => __( 'Performance', 'performance-lab' ),
+			'color' => 'blue',
+		),
+		'description' => '<p>' . esc_html__( 'Your site cache settings are configured correctly, helping to improve its performance.', 'performance-lab' ) . '</p>',
+		'actions'     => '',
+		'test'        => 'perflab_cch_cache_control_header_check',
+	);
+
+	$response = wp_remote_get(
+		home_url(),
+		array(
+			'headers'   => array( 'Content-Type' => 'text/html' ),
+			'sslverify' => false,
+		)
+	);
+
+	if ( is_wp_error( $response ) ) {
+		$result['label']       = __( 'Error checking cache settings', 'performance-lab' );
+		$result['status']      = 'recommended';
+		$result['description'] = '<p>' . wp_kses(
+			sprintf(
+				/* translators: %s is the error code */
+				__( 'There was an error while checking your site cache settings. Error code: <code>%s</code> and the following error message:', 'performance-lab' ),
```

Comment:
Here "site cache settings" can be confused potentially with someone setting up their page caching plugin. Perhaps better to be explicit what is being tested, even if it is technical:

```suggestion
				__( 'There was an error while checking your Cache-Control response header. Error code: <code>%s</code> and the following error message:', 'performance-lab' ),
```